skorch/tests/callbacks/test_training.py

```diff
@@ -1083,3 +1083,11 @@ def initialize_module(self, *args, **kwargs):
 
         assert save_params_mock.call_count == 1
         save_params_mock.assert_has_calls([call(f_mymodule='train_end_mymodule.pt')])
+
+
+class TestNoCallbacks(TestCheckpoint):
+    def test_no_callbacks(self, net_cls, data):
+        net = net_cls(callbacks="disable")
+        net.fit(*data)
```

Comment:
```suggestion
        net.initialize()
```
Let's not call `fit` here, since it's not necessary and it takes a bit of time. Consequently, there is also no more need for the `data` fixture.